mpris.c

```diff
@@ -149,6 +150,27 @@ static void add_metadata_item_string_list(mpv_handle *mpv, GVariantDict *dict,
     }
 }
 
+static void add_metadata_mb(mpv_handle *mpv, GVariantDict *dict,
+                                     const char *property, const char *tag)
+{
+    char *temp = mpv_get_property_string(mpv, property);
+
+    if (temp) {
+      /* Check for mutli-value ID3 v2.3 tag (backslash as separator) */
```

Comment:
The comment says backslash but the code has a forward slash? Which is it?

Reply:
the code is correct. I never have been able to get those names quite right..